src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -38,7 +38,25 @@
                             <span><img class="mx-1" src="~/icons/go-original.svg" width="40" alt="@Model.Review.Language"></span>
                             break;
                         case "java":
-                            <span><img class="mx-1" src="~/icons/java-original.svg" width="40" alt="@Model.Review.Language"></span>
+                            if(@Model.Review.Variant != null)
+                            {
+                                switch(@Model.Review.Variant.ToLower())
+                                {
+                                    case "spring":
+                                        <span><img class="mx-1" src="~/icons/java-spring-original.svg" width="40" alt="@Model.Review.Language @Model.Review.Variant"></span>
+                                        break;
+                                    case "android":
+                                        <span><img class="mx-1" src="~/icons/java-android-original.svg" width="40" alt="@Model.Review.Language @Model.Review.Variant"></span>
+                                        break;
+                                    default:
+                                        <span><img class="mx-1" src="~/icons/java-original.svg" width="40" alt="@Model.Review.Language"></span>
+                                        break;

```

Comment:
Eventually I would like to see this change to instead be a single line that uses code along the following lines:

if (variant != null && !variant.isEmpty()) {
    // set the img class property to be 'language-variant' (e.g. 'java-android')
} else {
    // set the img class property to be 'language' (e.g. 'java')
}

Then move all of the image file references out to CSS.